third_party/terraform/resources/resource_bigquery_table.go.erb

```diff
@@ -837,6 +900,23 @@ func expandTimePartitioning(configured interface{}) *bigquery.TimePartitioning {
 	return tp
 }
 
+<% unless version == 'ga' -%>
+func expandRangePartitioning(configured interface{}) *bigquery.RangePartitioning {
+	rpRaw := configured.([]interface{})[0].(map[string]interface{})
```

Comment:
I'd prefer if we were slightly safer here, in case len(configured) == 0 or for some reason we get [nil] - i know a couple of expanders in this file have it like this but I will probably go and fix them after this

Reply:
If we get nil, just return nil?

Reply:
yup! if we autogenerated this resource, it'd look more like: 

```
	l := configured.([]interface{})
	if len(l) == 0 || l[0] == nil {
		return nil, nil
	}
	rpRaw := l[0].(map[string]interface{})
```